README.md

```diff
@@ -4,11 +4,11 @@ I am a recent graduate from the University of Toronto with a background in softw
 
 ## Skills
```

Comment:
I am a recent graduate from the University of Toronto (CS) with a background in software development and data analytics. I am highly adaptive to new environments and always looking for opportunities to enhance my capabilities. Currently, I am working as a Junior Software and Data Engineer at Jarvis, where I work with my team in an agile environment to deliver high-quality software products. 
I believe that the software industry is moving at an incredibly rapid pace, with innovations and technologies being discovered every day. 
I am excited to pursue my two biggest passions in software and data engineering to join this ever-evolving community, utilizing big data to design products to make the world a better and infinitely more interesting place. Softs kills...

You have work exp focus soft skills, teamwork, colb, communication, leader..

Reply:
https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer
https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee